app/assets/javascripts/ace.js.coffee

```diff
@@ -1,4 +1 @@
-#= require ace/ace
-#= require ace/mode-javascript.js
-#= require ace/mode-markdown.js
-#= require ace/mode-coffee.js
+#= require ace-rails-ap
```

Comment:
Hmm that will not work for the docker images, we precompile the assets when the image is build and not at runtime. We could hardcode the setting in the image build process, but the simplest solution would be the to add mysql mode for everyone, even when not using the MysqlAgent. If we keep it configurable we also need to add a note in the manual installation guide to precompile the assets after modifying those settings.